packages/gen-ai-hub/src/orchestration/orchestration-completion-post-request.test.ts

```diff
@@ -46,10 +46,33 @@ describe('constructCompletionPostRequest()', () => {
     expect(completionPostRequest).toEqual(expectedCompletionPostRequest);
   });
 
+  it('with model configuration and empty template', async () => {
+    genaihubCompletionParameters.prompt.template = [];
```

Comment:
This is changing state for all tests, as it is globally scoped, either delete it or assign this value already when the variable is initialized.

Reply:
Never mind, since we reassign it beforeEach it should be fine.